app/build.gradle

```diff
@@ -40,13 +40,13 @@ android {
         debug {
             versionNameSuffix "Version: "
             resValue "string", "version", "${versionNameSuffix}${defaultConfig.versionName}"
-            resValue "string", "google_maps_key", google_key
+            //resValue "string", "google_maps_key", google_key
```

Comment:
We can delete these commented lines. With git, we can always check for them if we need to refer back.